src/Nancy.Testing/AssertExtensions.cs

```diff
@@ -93,7 +93,7 @@ public static AndConnector<QueryWrapper> ShouldBeOfClass(this QueryWrapper query
         /// </summary>
         public static AndConnector<NodeWrapper> ShouldContain(this NodeWrapper node, string contents, StringComparison comparisonType = StringComparison.Ordinal)
         {
-            Asserts.Contains(contents, node.InnerText, comparisonType);
+            Asserts.Contains(contents, node.InnerHtml, comparisonType);

```

Comment:
Is `InnerText` in CsQuery the text representation or the HTML representation?

In AngleSharp `InnerHtml` is `innerHTML` from the W3C specification, i.e., things like `<` (in text) will be represented as `&lt;`, whereas `TextContent` would represent the text only (leaving out the nodes), thus representing, e.g., `<`, literally.

According to the summary I believe `TextContent` would be more appropriate.